pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -882,19 +867,49 @@ func verifyStatus(ctx *domainContext, status *types.DomainStatus) {
 		// check if qemu processes has crashed
 		hasQemu := status.VirtualizationMode == types.HVM || status.VirtualizationMode == types.FML || status.IsContainer
 		if configActivate && status.Activated && hasQemu && !hyper.IsDeviceModelAlive(status.DomainId) {
-			errStr := fmt.Sprintf("verifyStatus(%s) qemu crashed",
+			//Have to check on the domain again before confirming qemu crash because of https://www.pivotaltracker.com/story/show/171962732
```

Comment:
The story URL doesn't belong in the source code.

Reply:
Removed.